protocol/x/perpetuals/types/types.go

```diff
@@ -120,9 +120,10 @@ type PerpetualsKeeper interface {
 	) []Perpetual
 }
 
+// OpenInterestDelta represents a (perpId, openInterestDelta) tuple.
 type OpenInterestDelta struct {
 	// The `Id` of the `Perpetual`.
 	PerpetualId uint32
 	// Delta of open interest (in base quantums).
-	BaseQuantumsDelta *big.Int
+	BaseQuantums *big.Int
```

Comment:
Nit change, felt like `BaseQuantums` is a unit and can be used to quantify both actual and delta values.